app/src/main/res/layout/hero_item_placeholder.xml

```diff
@@ -1,58 +1,60 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="@dimen/margin_default"
+    android:layout_marginTop="@dimen/margin_default"
+    android:layout_marginRight="@dimen/margin_default"
+    android:layout_marginBottom="@dimen/margin_default"
+    android:orientation="vertical"
+    tools:cardElevation="2dp"
+    tools:cardUseCompatPadding="true">
 
     <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent">
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
 
         <View
-                android:id="@+id/heroCardImage"
-                android:layout_width="@dimen/dp_0"
-                android:layout_height="@dimen/dp_200"
-                android:scaleType="fitXY"
-                android:background="@color/shimmer_background"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintTop_toTopOf="parent"
-                app:layout_constraintStart_toStartOf="parent" />
+            android:id="@+id/heroCardImage"
+            android:layout_width="@dimen/dp_0"
+            android:layout_height="@dimen/dp_200"
+            android:background="@color/shimmer_background"
+            android:scaleType="fitXY"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
 
         <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="horizontal"
-                app:layout_constraintTop_toBottomOf="@+id/heroCardImage">
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintTop_toBottomOf="@+id/heroCardImage">
 
             <View
-                    android:layout_weight="1"
-                    android:layout_gravity="center_vertical"
-                    android:layout_marginLeft="@dimen/margin_default"
-                    android:layout_marginRight="@dimen/margin_default"
-                    android:background="@color/shimmer_background"
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_20"/>
+                android:layout_width="@dimen/dp_0"
+                android:layout_height="@dimen/dp_20"
+                android:layout_gravity="center_vertical"
+                android:layout_marginLeft="@dimen/margin_default"
+                android:layout_marginRight="@dimen/margin_default"
+                android:layout_weight="1"
```

Comment:
Should be a grate idea create an integer file, to set this and other integer elements?

Reply:
done